gluon/gluon-setup-mode/files/lib/gluon/upgrade/setup-mode/invariant/005-setup-mode

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/lua
+
+local site = require 'gluon.site_config'
+local uci = require 'luci.model.uci'
+
+local c = uci.cursor()
```

Comment:
In all newer scripts, I've been using the following code:

    local uci = require('luci.model.uci').cursor()

and then `uci:save`, `uci:commit`... In my opinion using `uci:` instead of `c:` makes the code easier to read.